entrypoint.sh

```diff
@@ -99,12 +109,14 @@ http {
 }
 EOF
 
-# Initial certificate request
-letsencrypt certonly \
-  --domain ${DOMAIN} \
-  --authenticator standalone \
-  ${SERVER} \
-  --email "${EMAIL}" --agree-tos
+# Initial certificate request, but skip if cached
+if [ ! -f /etc/letsencrypt/live/${DOMAIN}/fullchain.pem ]; then
+  letsencrypt certonly \
+    --domain ${DOMAIN} \
+    --authenticator standalone \
+    ${SERVER} \
+    --email "${EMAIL}" --agree-tos
+fi
```

Comment:
These seem reasonable to me to check for. I'm assuming since the base image is controlled these only have to work for bash (no worries of super-POSIX compliance).